swagger.yaml

```diff
@@ -689,12 +696,12 @@ definitions:
     description: "The group model for the response body"
     type: object
     properties:
+      id:
+        description: "The group id"
+        type: string
       name:
         description: "The group name"
         type: string
-      description:
-        description: "A description of the group"
-        type: string
       precedence:
         description: "The priority of the group"
         type: integer
```

Comment:
Line 736 has an infinite recursion in the spec, but is not actually needed. We should delete this and update line 251 from `$ref: '#/definitions/UserGroups'` to `$ref: '#/definitions/GroupsList'`.

Reply:
I dont think that is right - having looking at the struct returned by this API endpoint it uses a different model? I could be wrong though. I will post my update so you can see what I think it should be.

Reply:
You are right they are different 👍